pkg/deploy/api/types.go

```diff
@@ -247,10 +247,10 @@ const (
 // These constants represent the various reasons for cancelling a deployment
 // or for a deployment being placed in a failed state
 const (
-	DeploymentCancelledByUser                 = "The deployment was cancelled by the user"
```

Comment:
Hrm - does the describer / event / other output still look ok?  Changing only in status is acceptable, but if the others are ugly that's a good reason to change.